src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/image.py

```diff
@@ -0,0 +1,50 @@
+from azure.cli.core.commands import LongRunningOperation
+from knack.util import CLIError
+from azure.mgmt.containerregistry.v2018_02_01_preview.models import (
+    ImportImageParameters,
+    ImportSource
+)
+from ._utils import (
+    validate_managed_registry,
+    get_resource_id_by_registry_name
+)
+
+IMPORT_NOT_SUPPORTED = "Image imports are only supported for managed registries."
+
+def acr_image_import(cmd,
+                     client,
+                     registry_name,
+                     image=None,
```

Comment:
IMO, `image` and `source_image` should be combined as the fully qualified image URL.

`az acr image import -n registryname --source-image sourceregistry.azurecr.io/myimage:v1 -t targetrepo:tag`

And `--source-registry-id` is only required if `sourceregistry` is in a different sub than `registryname`.

Another concern with using `--image` here is that, in your example, `--image` is used as a fully qualified image name, while the same parameter had been used under `repository` or `build-task` to represent `repository[:tag]` style.
`az acr image import -n registryname --image sourceregistry.azurecr.io/myimage:v1 -t targetrepo:tag`